.github/workflows/npm-test.yml

```diff
@@ -0,0 +1,14 @@
+name: npm-test
+run-name: ${{ github.actor }} is testing workflow
+
+on: [push, pull_request]
```

Comment:
For push you must inform the branch,  other wise it will run twice in pull request 

"Push" should run to master branch only

Reply:
I put it like this
{
on: 
push:
branches:[$default-branch] 
pull_request:
branches:[$default-branch]
}
 but I thought that leaving the push and pull as general wouldn't make much difference.

Reply:
hmm...fair
ok